lgc/patch/Patch.cpp

```diff
@@ -206,18 +200,24 @@ void Patch::addPasses(PipelineState *pipelineState, lgc::PassManager &passMgr, b
     // Extra optimizations after NGG primitive shader creation
     passMgr.addPass(AlwaysInlinerPass());
     passMgr.addPass(GlobalDCEPass());
-    FunctionPassManager fpm2;
-    fpm2.addPass(PromotePass());
-    fpm2.addPass(ADCEPass());
-    fpm2.addPass(InstCombinePass());
-    fpm2.addPass(SimplifyCFGPass());
-    passMgr.addPass(createModuleToFunctionPassAdaptor(std::move(fpm2)));
+    FunctionPassManager fpm;
+    fpm.addPass(PromotePass());
+    fpm.addPass(ADCEPass());
+    fpm.addPass(PatchBufferOp());
+    fpm.addPass(InstCombinePass());
```

Comment:
(Not related to your patch, but it is probably an omission that we do not cap the instcombine iterations here.)

Reply:
Kind of, yeah. Though it would be best if we just didn't need those caps in the first place.